docs/marketplace/dev/feature-walkthroughs/202108.0/persistence-acl-feature-walkthrough/configuration.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Configuration
+last_updated: Sep 14, 2021
+template: concept-topic-template
+---
+
+Persistence ACL feature based functionality on Propel behavior. It's possible to enable the feature in 2 ways:
+- Connect the feature to one or several tables in the database
+- Connect the feature to all tables in the database
+
+## Connect Persistence ACL feature to one or several tables
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" 
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          name="zed" 
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd" 
+          namespace="Orm\Zed\Merchant\Persistence"
+          package="src.Orm.Zed.Merchant.Persistence">
+    <table name="spy_merchant">
+      <behavior name="\Spryker\Zed\AclEntity\Persistence\Propel\Behavior\AclEntityBehavior"/>
+    </table>
+</database>
+```
+
+## Connect Persistence ACL feature to all tables
```

Comment:
Please mention smth like: in the following example _all_ the entities on the system will be handled by ACL. When you do such a configuration dont forget to use AllowList configuration (here should go a link) to exclude entities that are needed by ACL to function properly. Provide the list of entities that are needed (SpyRole, SpyEntiyRule, and all the other that are required).